script.js

```diff
@@ -0,0 +1,18 @@
+let saveEl = document.getElementById("save-el");
+let countEl = document.getElementById("count-el");
+let count = 0;
+
+function increment() {
+  count += 1;
+  countEl.textContent = count;
+}
+
+function save() {
+  let countStr = count + " - ";
+  saveEl.textContent += countStr;
+  countEl.textContent = 0;
+  count = 0;
+}
+
+event.preventDefault()
```

Comment:
What is the purpose of this line? event.preventDefault( ) is used when an element triggers an action by any specified event like "click" and you want to stop that action from happening. Suppose. you wish to submit a form through submit button and you have used form element. So, the default action of form submission is to reload the page. Here, you can stop this page reload by the preventDefault method.